Description: Install missing files
 The setup.py install is missing some files.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-02-09

--- python-oslo.vmware-0.9.0.orig/MANIFEST.in
+++ python-oslo.vmware-0.9.0/MANIFEST.in
@@ -3,4 +3,8 @@ include ChangeLog
 exclude .gitignore
 exclude .gitreview
 
+recursive-include oslo_vmware *.py
+recursive-include oslo_vmware/wsdl *
+recursive-include oslo/vmware *.py
+
 global-exclude *.pyc
\ No newline at end of file
